### DISCUSSION TITLE
how to correctly add elasticsearch

### DISCUSSION THREAD

**Discussion by moussaab-moulim** [vendurehq/vendure#3001] [General]
hello guys
i was. using the default search plugin and i am following the docs by using search to fetch my products
and i found to get my customfields i need to add elasticsearch
i never used elasticsearch
i deployed an instance in railway 
i added the host and username,password in vendure project

i ran it 

and suddenly i dont get any results in my search
is there a step that am i missing 
a synchronisation between my db and elasticsearch ? 
any help would be appreciated

PS: i am a noob when it comes to this subject 
thank you

**Comment by Wissaladd** [comment]
I am having the same problem I think we need more details about the search plugin in docs

**Reply by michaelbromley** [reply]
are either of you using multiple languages in your app?
